Jenkinsfile

```diff
@@ -0,0 +1,614 @@
+#!groovy
+
+def initializeEnvironment() {
+  env.DRIVER_DISPLAY_NAME = 'CassandraⓇ Java Driver'
+  env.DRIVER_METRIC_TYPE = 'oss'
+  if (env.GIT_URL.contains('riptano/java-driver')) {
+    env.DRIVER_DISPLAY_NAME = 'private ' + env.DRIVER_DISPLAY_NAME
+    env.DRIVER_METRIC_TYPE = 'oss-private'
+  } else if (env.GIT_URL.contains('java-dse-driver')) {
+    env.DRIVER_DISPLAY_NAME = 'DSE Java Driver'
+    env.DRIVER_METRIC_TYPE = 'dse'
+  }
+
+  env.GIT_SHA = "${env.GIT_COMMIT.take(7)}"
+  env.GITHUB_PROJECT_URL = "https://${GIT_URL.replaceFirst(/(git@|http:\/\/|https:\/\/)/, '').replace(':', '/').replace('.git', '')}"
+  env.GITHUB_BRANCH_URL = "${GITHUB_PROJECT_URL}/tree/${env.BRANCH_NAME}"
+  env.GITHUB_COMMIT_URL = "${GITHUB_PROJECT_URL}/commit/${env.GIT_COMMIT}"
+
+  env.MAVEN_HOME = "${env.HOME}/.mvn/apache-maven-3.3.9"
+  env.PATH = "${env.MAVEN_HOME}/bin:${env.PATH}"
+  env.JAVA_HOME = sh(label: 'Get JAVA_HOME',script: '''#!/bin/bash -lex
+    . ${JABBA_SHELL}
+    jabba which ${JABBA_VERSION}''', returnStdout: true).trim()
+  env.JAVA8_HOME = sh(label: 'Get JAVA8_HOME',script: '''#!/bin/bash -lex
+    . ${JABBA_SHELL}
+    jabba which 1.8''', returnStdout: true).trim()
+
+  sh label: 'Download Apache CassandraⓇ or DataStax Enterprise',script: '''#!/bin/bash -lex
+    . ${JABBA_SHELL}
+    jabba use ${JABBA_VERSION}
+    . ${CCM_ENVIRONMENT_SHELL} ${SERVER_VERSION}
+  '''
+
+  sh label: 'Display Java and environment information',script: '''#!/bin/bash -le
```

Comment:
I confirm that this step in Blue Ocean is much less verbose than the 2 previous ones that are still using -x